R/call_derivation.R

```diff
@@ -3,7 +3,7 @@
 #' Call a single derivation multiple times with some parameters/arguments being fixed across
 #' iterations and others varying.
 #'
-#' @param dataset The input dataset
+#' @param dataset  `r roxygen_param_dataset()`
```

Comment:
@pharmaverse/admiral Is *just* "Input dataset" the appropriate description, or should all entries mention something about expected variables?

Reply:
I think in this case just "Input dataset" is OK.